src/mainwindow/mainwindow.cpp

```diff
@@ -229,12 +230,28 @@ void MainWindow::setupMainMenu()
     helpMenu->addSeparator();
 
     // opening reference window
-    QAction *pReferenceAction = helpMenu->addAction("&Reference Assistant...", this, &MainWindow::onReferenceTriggered, Qt::CTRL + Qt::Key_F1);
+    QAction *pReferenceAction = helpMenu->addAction("&Reference Assistant...",
```

Comment:
Why these spaces appeared. Remove them if they are not needed.

Reply:
done